Week2/homework/js-exercises/ex2.js

```diff
@@ -0,0 +1,14 @@
+//Create a for loop, that iterates from 0 to 20//[i] is [21] start from 0
+for (i = 0; i <= 20; i++) {
+  //Create a conditional statement that checks if the value of the counter variable is odd or even
+  if (i % 2 === 0) {
+    console.log(`The number ${i} is even`);
+  } else console.log(`The number ${i} is odd`);
```

Comment:
nice, it's a good practice to always include the `{` and the `}`. In order to avoid silly mistakes

So the else would look like this then:

```
else {
 console.log(`The number ${i} is odd`);
}
```